convert/struct.go

```diff
@@ -0,0 +1,40 @@
+package convert
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/spdx/tools-golang/spdx/common"
+)
+
+// FromPtr accepts a document or a document pointer and returns the direct struct reference
+func FromPtr(doc common.Document) common.Document {
```

Comment:
does `FromPtr` and `Describe` have to be public?

Reply:
Yes, they're used by different packages